tests/integration/pause.tcl

```diff
@@ -0,0 +1,64 @@
+source tests/support/benchmark.tcl
+
+start_server {tags {"external:skip"}} {
+  set master_host [srv 0 host]
+  set master_port [srv 0 port]
+
+  test {Test client-pause-write-during-oom} {
+    # Scenario:
+    # When client-pause-write-during-oom is configured, if OOM and available pending 
+    # lazy-free jobs in background, then (clients are paused and) eviction is 
+    # suspended. The test will:
+    # - Fill up DB with many small items and configures tight maxmemory
+    # - FLUSHALL ASYNC  (slow background operation by lazyfree thread)    
+    # - Fill up DB with many big items (quick operation by main thread)
+    #  
+    # It is expected that filling up DB for the second time will be quicker than the
+    # FLUSHALL ASYNC release and will cause to OOM, and if client-pause-write-during-oom
+    # is configured then there won't be any eviction or only few 
+
+    set FEATURE_DISABLED no
+    set FEATURE_ENABLED yes
+
+    # Prepare cmd2 in advance in order to run as fast as possible after flushall
+    set cmd1 [redisbenchmark $master_host $master_port " -t set -n 990000 -r 100000000 -d 1 -P 150"]
+    set cmd2 [redisbenchmark $master_host $master_port " -t set -n 2200 -r 100000000 -d 20000 -P 20"]
```

Comment:
i don't think we wanna use redis-benchmark to populate the database for tests (unless we test redis-benchmark).
we have the TCL based `populate` function, and we can improve that if needed.

Reply:
I think the missing features and performance between redis-benchamark and populate is too big.